_gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md

```diff
@@ -0,0 +1,36 @@
+---
+project: ESCAPE-OSSR
+title: Repository as a Service
+author: Sudersan S
+date: 04.08.2022
+year: 2022
+layout: blog_post
+logo: OSSR-logo.png
+intro: |
+This blog entails my progress, learning and experience acquired during the GSoC period.
```

Comment:
```suggestion
 This blog entails my progress, learning and experience acquired during the GSoC period.
```

Reply:
You need to add a space at least at the beginning of the intro to be interpreted correctly